addons/account/models/account_move_line.py

```diff
@@ -2044,15 +2063,38 @@ def _prepare_reconciliation_single_partial(self, debit_values, credit_values, sh
 
             # Compute the partial amount expressed in company currency.
             partial_amount = min_recon_amount
+            if debit_rate and debit_currency != company_currency \
+               and company_currency.compare_amounts(recon_debit_amount, max_recon_amount):
+                recon_debit_amount_range = get_amount_range_after_rate(
+                    currency_from=company_currency,
+                    currency_to=debit_currency,
+                    amount=recon_debit_amount,
+                    rate=debit_rate,
+                )
+                if amount_in_range(debit_currency, max_recon_amount * debit_rate, recon_debit_amount_range):
+                    partial_amount = max_recon_amount
+            if credit_rate and credit_currency != company_currency \
+               and company_currency.compare_amounts(recon_credit_amount, max_recon_amount):
+                recon_credit_amount_range = get_amount_range_after_rate(
+                    currency_from=company_currency,
+                    currency_to=credit_currency,
+                    amount=recon_credit_amount,
+                    rate=credit_rate,
+                   )
+                if amount_in_range(credit_currency, max_recon_amount * credit_rate, recon_credit_amount_range):
+                    partial_amount = max_recon_amount
```

Comment:
That part is wrong.
There is only a single field to express the amount in company currency on the partial. It has to stay the minimum^^